src/modules/i18n/containers/__tests__/I18nProvider.spec.js

```diff
@@ -121,23 +127,24 @@ describe.skip('I18nProvider', () => {
       const originalGetSelectedFonts = I18nProvider.getSelectedFonts
       // $FlowFixMe
       I18nProvider.getSelectedFonts = jest.fn(I18nProvider.getSelectedFonts)
-      i18n.changeLanguage = jest.fn()
+      i18n.changeLanguage = jest.fn(i18next.changeLanguage)
```

Comment:
Do we have to use `i18next.changeLanguage` here? Couldn't we use a manual defined mock function or `jest.fn(i18n.changeLanguage)`?